my-app/src/components/login/login.js

```diff
@@ -1,73 +1,74 @@
+/* eslint-disable react/jsx-filename-extension */
```

Comment:
Errors should be fixed if possible, instead of ignored. For example the JSX filename issue is an easy fix by just changing the file extension. Some of the other errors are trickier to fix but most of them should be fixed instead of ignored because there are ways to fix without breaking our code